pyproject.toml

```
@@ -24,7 +24,7 @@ include = ["AUTHORS.rst", "CHANGES.rst", "CONTRIBUTING.rst", "CITATION"]
 
 [tool.poetry.dependencies]
 python = ">=3.8"
-numpy = ">=1.18"
+numpy = ">=1.18, <=1.24"
```

```suggestion
numpy = ">=1.18, <2.0"
```
should be sufficient for oktopus, at least it is still importing and passing tests with 1.26.4